src/Microsoft.AspNet.Antiforgery/DefaultAntiforgery.cs

```diff
@@ -26,34 +25,40 @@ public class DefaultAntiforgery : IAntiforgery
             IOptions<AntiforgeryOptions> antiforgeryOptionsAccessor,
             IAntiforgeryTokenGenerator tokenGenerator,
             IAntiforgeryTokenSerializer tokenSerializer,
-            IAntiforgeryTokenStore tokenStore,
-            HtmlEncoder htmlEncoder)
+            IAntiforgeryTokenStore tokenStore)
         {
             _options = antiforgeryOptionsAccessor.Value;
             _tokenGenerator = tokenGenerator;
             _tokenSerializer = tokenSerializer;
             _tokenStore = tokenStore;
-            _htmlEncoder = htmlEncoder;
         }
 
         /// <inheritdoc />
-        public string GetHtml(HttpContext context)
+        public void GetHtml(HttpContext context, IHtmlContentBuilder content)
```

Comment:
Yeah, having looked at the MVC reaction PR, I really don't get why this is better than just writing a private class here and returning it.